game/scripts/npc/items/item_radiance_2.txt

```diff
@@ -1,84 +1,84 @@
 "DOTAItems"
 {
-	//=================================================================================================================

```

Comment:
If people aren't careful with their line endings this shit can happen. In SublimeText3 you go to View > Line Endings > and make sure it says Unix, not Windows. Also you can enable EditorConfig in your SublimeText3. Furthermore there's also a way to make Git convert them automatically. I have no fucking idea. The best bet is just to check everytime you edit one. It's a bitch aint it?